TCC.Lib/Benchmark/BenchmarkOptionHelper.cs

```diff
@@ -33,7 +39,7 @@ public static PasswordOption GenerateDecompressPasswordOption(PasswordMode passw
             }
         }
 
-        public static async Task<PasswordOption> GenerateCompressPassswordOption(PasswordMode passwordMode, string keysFolder)
+        public async Task<PasswordOption> GenerateCompressPassswordOption(PasswordMode passwordMode, string keysFolder)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this method into two, one handling parameters check and the other handling the asynchronous code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS4457)